packages/contracts/contracts/voting/majority/MajorityVotingBase.sol

```diff
@@ -99,17 +134,17 @@ abstract contract MajorityVotingBase is
     /// @notice A container for the proposal parameters at the time of proposal creation.
     /// @param votingMode A parameter to select the vote mode.
     /// @param supportThreshold The support threshold value.
-    /// @param minParticipation The minimum participation value.
     /// @param startDate The start date of the proposal vote.
     /// @param endDate The end date of the proposal vote.
     /// @param snapshotBlock The number of the block prior to the proposal creation.
+    /// @param minVotingPower The minimum voting power needed.
     struct ProposalParameters {
         VotingMode votingMode;
         uint64 supportThreshold;
```

Comment:
> * Doesn't support threshold need the exact same treatment as minParticipation?
> * I would move this variable, below, right next to minVotingPower

No, it must remain a ratio so that we can do
```solidity
/// @inheritdoc IMajorityVoting
function isSupportThresholdReached(uint256 _proposalId) public view virtual returns (bool) {
    Proposal storage proposal_ = proposals[_proposalId];
    return
        (RATIO_BASE - proposal_.parameters.supportThreshold) * proposal_.tally.yes >
        proposal_.parameters.supportThreshold * proposal_.tally.no;
}
```
Accordingly, it must stay where it is to make use of tight-variable packing with the other `uint64` values.

> Why renaming this? Doesn't minParticipation still imply a voting power?

I don't understand what you mean. `minParticipation` is a relative value (a percentage). `minVotingPower` is an absolute value.